tech-far-hub/src/components/multi-step.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+
+import { Alert, Card, CardBody } from "@trussworks/react-uswds";
+
+export const MultiStep = ({ children }: { children: React.ReactElement }) => {
+  const errorMessage = (
+    <Alert type="warning" headingLevel="h3" heading="Error in process list">
+      A MultiStep list must be an ordered list.
+      <a href="https://www.markdownguide.org/basic-syntax/#ordered-lists">See the markdown guide for examples.</a>
+    </Alert>
+  );
+
+  if (children.type !== "ol") {
+    return errorMessage;
+  }
+
+  const elementFilter = (item: React.ReactNode) => item && typeof item === "object" && "type" in item;
+  let items = children.props.children.filter(elementFilter);
+
+  if (
+    !items.every((item: React.ReactNode) => item && typeof item === "object" && "type" in item && item.type == "li")
+  ) {
+    return errorMessage;
+  }
+  const [isVisible, setIsVisible] = React.useState([true, ...Array(items.length).fill(false)]);
```

Comment:
really clear state management. I'm assuming the first card is visible, while the rest are invisible. Anytime an item is clicked, make that one visible and the rest invisible. Awesome variables naming. This is a great example of clear code, with no comments!

Reply:
thank you <3 that is my aspiration!